.github/workflows/build-latest.yaml

```diff
@@ -0,0 +1,197 @@
+name: build-latest
+on:
+#  workflow_dispatch:
+#    inputs:
+#      distro:
+#        description: Base image distro
+#        required: true
+#        default: debian
+#      imageVersion:
+#        description: Base distro image version/release
+#        required: true
+#        default: bullseye
+#      imageVariant:
+#        description: Base image variant
+#        required: true
+#        default: slim
```

Comment:
This is commented for now, but can be useful if we have a use case to do an ad-hoc cloud build via GH Action.

For example, if the recipe can handle different base image. We can try to do a build from GH UI.

Reply:
Not much experience with actions